source/headers.ts

```diff
@@ -107,6 +143,36 @@ export const setDraft7Headers = (
 	)
 }
 
+/**
+ * Sets `RateLimit` & `RateLimit-Policy` headers based on the eighth draft of the spec.
+ * See https://datatracker.ietf.org/doc/html/draft-ietf-httpapi-ratelimit-headers-08.
+ *
+ * @param response {Response} - The express response object to set headers on.
+ * @param info {RateLimitInfo} - The rate limit info, used to set the headers.
+ * @param windowMs {number} - The window length.
+ * @param key {string} - The unique string identifying the client.
+ */
+export const setDraft8Headers = (
+	response: Response,
+	info: RateLimitInfo,
+	windowMs: number,
+	key: string,
+): void => {
+	if (response.headersSent) return
+
+	const windowSeconds = Math.ceil(windowMs / 1000)
+	const resetSeconds = getResetSeconds(info.resetTime, windowMs)
+	const duration = getDurationInWords(windowMs)
+	const partitionKey = getPartitionKey(key)
+
+	const name = `rl-${info.limit}-in-${duration}`
+	const policy = `q=${info.limit}; w=${windowSeconds}; pk=:${partitionKey}:`
+	const header = `r=${info.remaining}; t=${resetSeconds!}`
+
+	response.setHeader('RateLimit-Policy', `"${name}"; ${policy}`)
+	response.setHeader('RateLimit', `"${name}"; ${header}`)
```

Comment:
I think this will overwrite the existing one if two instances set headers - we should instead append in that case, with an array of values.

Reply:
`res.append()` is a good solution, but we'll need to bump the express peer dependency from 4 to 4.11 in `package.json` - http://expressjs.com/en/4x/api.html#res.append

(Honestly we should probably double check see if there are any other apis we're using that require versions of Express newer than 4.0.0)

Reply:
Done